### PR TITLE
DEVX-2142: Update deprecated functions (register, pipeInput, and read…

### PR DESCRIPTION
**Pull request by ybyzek**
…Output)

**Description**: address warning during compile

```
> Task :compileTestJava
/Users/yeva/git/demos-common/src/test/java/io/confluent/demos/common/wiki/WikipediaActivityMonitorTest.java:55: warning: [deprecation] register(String,Schema) in SchemaRegistryClient has been deprecated
            schemaRegistryClient.register(topic + "-value", s);
                                ^
/Users/yeva/git/demos-common/src/test/java/io/confluent/demos/common/wiki/WikipediaActivityMonitorTest.java:77: warning: [deprecation] pipeInput(ConsumerRecord<byte[],byte[]>) in TopologyTestDriver has been deprecated
            topologyTestDriver.pipeInput(consumerRecord);
                              ^
/Users/yeva/git/demos-common/src/test/java/io/confluent/demos/common/wiki/WikipediaActivityMonitorTest.java:86: warning: [deprecation] <K,V>readOutput(String,Deserializer<K>,Deserializer<V>) in TopologyTestDriver has been deprecated
            final ProducerRecord<K, V> record = topologyTestDriver.readOutput(topic, keyDeserializer, valueDeserializer);
                                                                  ^
  where K,V are type-variables:
    K extends Object declared in method <K,V>readOutput(String,Deserializer<K>,Deserializer<V>)
    V extends Object declared in method <K,V>readOutput(String,Deserializer<K>,Deserializer<V>)
3 warnings
```

**Validation completed**:

Added the following to build.gradle and ran `make clean test`

```
tasks.withType(JavaCompile) {
    options.compilerArgs << '-Xlint:unchecked'
    options.deprecation = true
}
```

**Request of reviewers**:

- [x] Review PR
- [x] Validate PR code changes

### CODE REVIEW

**Comment by awalther28** [comment]
Overall looks good. `make clean test` pasts without the warnings. Given the removals, we don't need the following imports

**Comment by ybyzek** [comment]
> Overall looks good. `make clean test` pasts without the warnings. Given the removals, we don't need the following imports
> 
> ```
> import org.apache.kafka.clients.consumer.ConsumerRecord;
> import org.apache.kafka.clients.producer.ProducerRecord;
> import org.apache.kafka.common.record.TimestampType;
> ```
> 
> @ybyzek can we remove these please if they aren't going to be used anymore?

@awalther28 great catch, thank you.  Removed in https://github.com/confluentinc/demos-common/pull/8/commits/467eb68b65fe7f99f142caebe7caecb302f7ecbe

**Review by awalther28 — APPROVED** [review]
Thanks @ybyzek LGTM

File: src/test/java/io/confluent/demos/common/wiki/WikipediaActivityMonitorTest.java [hunk 1]
```diff
@@ -8,13 +9,12 @@
...
-import org.apache.kafka.common.record.TimestampType;
 import org.apache.kafka.common.serialization.*;
 import org.apache.kafka.streams.KeyValue;
 import org.apache.kafka.streams.StreamsBuilder;
 import org.apache.kafka.streams.TopologyTestDriver;
+import org.apache.kafka.streams.TestInputTopic;
+import org.apache.kafka.streams.TestOutputTopic;
 import org.junit.Test;
 
 import java.io.FileInputStream;
```
Comment (rspurgeon) [inline src/test/java/io/confluent/demos/common/wiki/WikipediaActivityMonitorTest.java:20]: Not related to this change but this import is unused as well

**Review by rspurgeon — APPROVED** [review]
LGTM left an optional minor comment

File: src/test/java/io/confluent/demos/common/wiki/WikipediaActivityMonitorTest.java (hunk 1 above)
Comment (ybyzek) [reply src/test/java/io/confluent/demos/common/wiki/WikipediaActivityMonitorTest.java]: Thanks. Removed...and validated.

**Pull request merged**